Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_test.py

```diff
@@ -2178,3 +2195,65 @@ def test_build_common_files_query(self):
             )
             actual = cu.build_common_files_query()
             assert actual == expected
+
+
+def test_get_machine_users_command(mocker):
```

Comment:
Please add a “Given, When, Then” to the test, example can be found in the other tests in the file

Reply:
Added given, when then